src/contracts/test/GPv2EncodingTestInterface.sol

```diff
@@ -35,15 +35,19 @@ contract GPv2EncodingTestInterface {
         view
         returns (
             GPv2Encoding.Trade[] memory trades,
+            uint256 tradeCount,
```

Comment:
super nit-picky, but should this be:
```suggestion
            uint256 tradeCount,
            GPv2Encoding.Trade[] memory trades,
```

So that it is more consistent with the other trade method `tradeCountTest` and the order in which the data is encoded?

Reply:
This helps you to switch to object destructuring :point_up: .